esipy/cache.py

```diff
@@ -1,4 +1,4 @@
-# -*- encoding: utf-8 -*-
+'''-*- encoding: utf-8 -*-'''

```

Comment:
This is wrong: this line is not a "comment" but define how the file is encoded for python and should start with `#`
https://www.python.org/dev/peps/pep-0263/